src/local-dem-manager.ts

```diff
@@ -92,24 +75,10 @@ export class LocalDemManager implements DemManager {
     return this.tileCache.get(
       url,
       (_, childAbortController) => {
-        const options: RequestInit = {
-          signal: childAbortController.signal,
-        };
         timer?.fetchTile(url);
-        const mark = timer?.marker("fetch");
```

Comment:
This metrics gathering is missing from the getTile implementation, can we add it back in outside of the getTile call, something like:

```ts
const mark = timer?.mark("fetch");
return withTimeout(
  this.timeoutMs,
  this.getTile(url, childAbortController).finally(() => mark?.()),
  childAbortController,
);
```

Reply:
Sure, wasn't entirely sure how to solve this...

Reply:
Done in edfe7d1